### PR TITLE
feat(git): Change Commit Log to use Lazygit

### PR DESCRIPTION
**Pull request by georgeguimaraes**
I found that using Lazygit Commit Log is much more helpful. You can have all the Lazygit usefulness here.

I'm using it to browse the last commits, and then browse again the files that were changed in those commits, and using the `e` shortcut to open the file in the same Neovim instance.

### CODE REVIEW

**Comment by dpetka2001** [comment]
Really nice change. Hope it gets merged!! :smile:

**Review by rubiin — APPROVED** [review]

**Comment by rubiin** [comment]
I am using lazygit  too so this makes sense to me as well to use it for most things :)

File: lua/lazyvim/plugins/editor.lua [hunk 1]
```diff
@@ -158,7 +158,6 @@ return {
       { "<leader>fr", "<cmd>Telescope oldfiles<cr>", desc = "Recent" },
       { "<leader>fR", LazyVim.telescope("oldfiles", { cwd = vim.uv.cwd() }), desc = "Recent (cwd)" },
       -- git
-      { "<leader>gc", "<cmd>Telescope git_commits<CR>", desc = "Commits" },
```
Comment (jyuan0) [inline lua/lazyvim/plugins/editor.lua:161]: If you'd like to not remove a mapping, `<leader>gl` is free and makes sense to me.

Comment (dpetka2001) [reply lua/lazyvim/plugins/editor.lua:161]: But they do the same thing. They show the commits. It doesn't make sense to have 2 different mappings for 2 plugins that do the same thing essentially (to me personally at least).

Comment (rubiin) [reply lua/lazyvim/plugins/editor.lua:161]: > But they do the same thing. They show the commits. It doesn't make sense to have 2 different mappings for 2 plugins that do the same thing essentially (to me personally at least).

I agree.

**Comment by folke** [comment]
I kept the original `<leader>gc` and added a `<leader>gl` for LazyGit Log

**Pull request merged**